src/Microsoft.DotNet.Build.Tasks/Targets/coreAssembly.rsp

```diff
@@ -0,0 +1,2 @@
+﻿# CSC requires the runtimemetadataversion flag when building a core assembly (containing System.Object)
+/runtimemetadataversion:v4.0.30319
```

Comment:
Last time I checked there was no good detection. I think we were considering using the msbuild version number.